dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/java_gateway.py

```diff
@@ -63,3 +63,130 @@ def gateway_result_checker(
     ):
         raise PyDSJavaGatewayException("Get result state not success.")
     return result
+
+
+java_gateway = launch_gateway()
+
+
+def get_datasource_info(name):
```

Comment:
We should better add some type hint for parameter.
```suggestion
def get_datasource_info(name: str):
```